src/sparc/client/client.py

```diff
@@ -27,15 +57,37 @@ def __init__(self, config_file="config/config.ini", connect=True) -> None:
             logging.debug(str(config))
         current_config = config["global"]["default_profile"]
 
-        logging.debug(str(current_config))
+        # logging.debug(str(current_config))
```

Comment:
Generally commented code is considered "code smell"
In our repositories we avoid them or if you really want to keep it, we add a ``NOTE:`` that justifies it